modules/oe_content_featured_media_field/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenEuropa Content Featured Media Field
+
+This module provides the Featured media field type containing a media reference field and a caption textfield.
+It includes an autocomplete widget, and an entity browser widget for the field type.
```

Comment:
No comma between widget and and.

Reply:
![image](https://user-images.githubusercontent.com/22908988/84901435-c2a96900-b0ab-11ea-8d29-de547e85c9ea.png)
😁

Reply:
It does not separate two clauses. It's an enumeration: it includes and autocomplete widget and an entity browser. 

Mary ate an apple, and a pear ?!?!!?